packages/ui/src/molecules/Accordion/AccordionItem.tsx

```diff
@@ -69,4 +83,11 @@ export function useAccordionItem() {
   return context
 }
 
+/**
+ * This is only being exported to make it easier to use in Storybook.
+ * **DON'T** import this directly to use this component, use the default export
+ * instead.
+ */
+ export const StorybookAccordionItem = AccordionItem as FC<Props>
```

Comment:
Just an extra space.
```suggestion
export const StorybookAccordionItem = AccordionItem as FC<Props>
```